web/netlify.toml

```diff
@@ -9,6 +9,7 @@ NODE_VERSION = "20"
 VITE_APP_OAUTH_API_ROOT = "https://api-staging.dandiarchive.org/oauth/"
 VITE_APP_OAUTH_CLIENT_ID = "Dk0zosgt1GAAKfN8LT4STJmLJXwMDPbYWYzfNtAl"
 VITE_APP_DANDI_API_ROOT = "https://api-staging.dandiarchive.org/api/"
+VITE_APP_DOI_SERVER = "https://handle.stage.datacite.org/"
```

Comment:
FTR note to myself: somehow Findable DOIs gather this URL for them on test fabric, e.g. here is Findable and draft to compare:

![image](https://github.com/user-attachments/assets/8296bfef-750b-41a0-9bbe-5013bcc0aaaa)

Reply:
yes either registered or findable will generate that handle, but right now the handle doesn't do what the doi version does.